projects/helper/chain/radixdlt.js

```diff
@@ -40,45 +40,74 @@ async function queryAddresses({ addresses = [], }) {
   return items
 }
 
-async function queryLsus(addresses = []) {
-  // Receives a list of lsu addresses and returns the redemption value in XRD
-
+async function queryLiquidStakeUnitDetails(addresses = []) {
+  /**
+   * queryLsus takes a list of addresses as argument.
+   * The return value is a dictionary with the lsu address and it's valuation in xrd.
+   * Example response:
+   *
+   * {
+      resource_rdxabc1: {
```

Comment:
Let's use real mainnet addresses, eg 

* `resource_rdx1tkfxrsffdlh82fjxjwpgrgrgcmc7cfe0sy99c7vm6gsnujelupglnj` belonging to `validator_rdx1s0g5uuw3a7ad7akueetzq5lpejzp9uw5glv2qnflvymgendvepgduj`
* `resource_rdx1t5x9fx6vjyk5x77vtqtzchxdlk5p9hgp8vpz8v3s0dzdu0dmps58pu` belonging to `validator_rdx1s0lz5v68gtqwswu7lrx9yrjte4ts0l2saphmplsz68nsv2aux0xvfq`

Reply:
works for me